File: UnionSwiss.Api/UnionSwiss.Domain/Model/Entity/TaxBracket.cs
using System;
using UnionSwiss.Domain.Model.Entity.Interface;

namespace UnionSwiss.Domain.Model.Entity
{
    public class TaxBracket:IEntity<long>
    {
        public long Id { get; set; }
        public long TaxYearId { get; set; }
        public int MinQualifyingValue { get; set; }
        public int MaxQualifyingValue { get; set; }
  
        public int BaseTaxValue { get; set; }
        public Decimal IncrementMultiplier { get; set; }


    }
}

File: UnionSwiss.Web/src/app/shared/types/taxBracket.ts

export class TaxBracket {
    id: number;
    taxYearId: number;
    minQualifyingValue: number;
    maxQualifyingValue: number;
    baseTaxValue: number;
    incrementMultiplier: number;
    constructor() {
        this.id = 0;
        this.taxYearId = 0;
        this.minQualifyingValue = 0;
        this.maxQualifyingValue = 0;
        this.baseTaxValue = 0;
        this.incrementMultiplier = 0;
    }
};
